### PR TITLE
Mobileview

### PR DESCRIPTION
**Pull request by saisitaram2000**
added inc dec cart buttons

### CODE REVIEW

File: frontend/src/components/Cart.jsx [hunk 1]
```diff
@@ -43,12 +45,12 @@ class Cart extends Component {
                     <div className="cart-item-middle">
                         <div className="cart-item-middle-name">{item.name}</div>
                         <div className="cart-item-middle-quantity">
-                            <div>
+                           {/* <div>
                                 <svg className="cart-item-servings-icon">
                                     <use href="/images/icons.svg#icon-man" />
```
Comment (sharshach) [inline frontend/src/components/Cart.jsx:50]: Remove Commented code

File: frontend/src/components/Items/Item.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    max-width: 100%;
+}
+.item-details{
+    position: relative;
+    display: flex;
+    flex-direction: column;
+}
+.item-details-name{
+    font-weight: 700;
+    font-size: 2.75rem;
```
Comment (sharshach) [inline frontend/src/components/Items/Item.css:23]: Give in px (use same formattings all over code)

File: frontend/src/components/Items/Item.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    flex-direction: row;
+    justify-content: center;
+}
+.item-details-price{
+    font-size: 18px;
+   font-weight: 500;
+   text-align: left;
+}
+.item-details-value{
+    text-align: center;
```
Comment (sharshach) [inline frontend/src/components/Items/Item.css:38]: Needed?

File: frontend/src/components/Items/Item.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+}
+.item-details-inc-dec-cart-buttons{
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    flex-wrap: wrap;
+}
+.item-btn-addtocart {
+    line-height: 4;
```
Comment (sharshach) [inline frontend/src/components/Items/Item.css:51]: Needed?

File: frontend/src/components/SampleItem.jsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import React,{useState,useEffect} from 'react';
```
Comment (sharshach) [inline frontend/src/components/SampleItem.jsx:1]: Delete the file if you are not using

File: frontend/src/components/Util/MyButton.jsx [hunk 6]
```diff
@@ -3,7 +3,7 @@ import Button from '@material-ui/core/Button';
 
 export default class MyButton extends Component {
     render() {
-        const { color,variant, href,icon } = this.props;
+        const { color,variant, href,icon,data} = this.props;
```
Comment (sharshach) [inline frontend/src/components/Util/MyButton.jsx:6]: Space after   ,    (before data)

**Review by sharshach — CHANGES_REQUESTED** [review]
Change these

**Review by journeynitdgp — APPROVED** [review]

**Review by sharshach — APPROVED** [review]

**Pull request merged**